src/particle/pdgid/__init__.py

```diff
@@ -106,6 +108,8 @@
 from .pdgid import PDGID
 
 __all__ = (
+    "PDGID",
```

Comment:
I had added PDGID to the list below (you got it when rebasing). This is now a duplicate.

Reply:
Okay!